doc/developers.md

```diff
@@ -99,7 +99,9 @@ spdlog::debug("Send help");
 ```
 
 ### Compiling on UCL's Myriad cluster
+
 <details>
+
```

Comment:
On the deployed version we have...

![Screenshot 2022-12-13 at 11 24 43](https://user-images.githubusercontent.com/1836192/207315923-b5941e53-9357-4766-b8bc-f7e95e3fdcce.png)